metadata-ingestion/src/datahub/ingestion/source/kafka.py

```diff
@@ -356,9 +384,19 @@ def _extract_record(
         browse_path = BrowsePathsClass([browse_path_str])
         dataset_snapshot.aspects.append(browse_path)
 
-        custom_props = self.build_custom_properties(
-            topic, topic_detail, extra_topic_config
-        )
+        # build custom properties for topic, schema properties may be added as needed
+        custom_props: Dict[str, str] = {}
+        if len(topic) != 0:
+            custom_props = self.build_custom_properties(
+                topic, topic_detail, extra_topic_config
+            )
+            schemaName: Optional[
```

Comment:
Can you change the variable to use the python naming convention?:
```
schema_name: Optional[
```

Reply:
done